lib/codelyzer/README.md

```diff
@@ -1,24 +1,24 @@
 <h1 align="center">
-   code style - TSLint codelyzer
+   code style - codelyzer
 </h1>
 
 ## About
 
-This sub-package hosts the [TSLint codelyzer](https://github.com/mgechev/codelyzer) configuration.
+This sub-package hosts the [codelyzer](https://github.com/mgechev/codelyzer) configuration.
 
 To be able to use this config, you need to install `codelyzer`.
 
 The following versions are available (based on **codelyzer** version):
 
--   4.5.x - `@nationalbankbelgium/code-style/tslint-codelyzer/4.5.x`
+-   4.5.x - `@nationalbankbelgium/code-style/lib/codelyzer/4.5.x`
```

Comment:
Small typo: It should be: `@nationalbankbelgium/code-style/codelyzer/4.5.x`

Reply:
As I mentioned in my last comment, the paths should contain the `/lib` folder otherwise the config cannot be found. The reason is that we have all these configs under the `/lib` folder. Therefore I also mentioned that we could move the folders one level up so they are accessible directly at the root so the path would indeed be: "@nationalbankbelgium/code-style/codelyzer/4.5.x"

Reply:
Never mind, I didn't test this properly :(